### PR TITLE
Abuse c on inetnum inet6num autnum

### PR DESCRIPTION
**Pull request by sanderbuskens**

### CODE REVIEW

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AutNumAuthSpec.groovy [hunk 1]
```diff
@@ -4206,4 +4206,46 @@ class AutNumAuthSpec extends BaseQueryUpdateSpec {
...
+                export:         to AS1 announce AS2
+                mp-import:      afi ipv6.unicast from AS1 accept ANY
+                mp-export:      afi ipv6.unicast to AS1 announce AS2
+                org:            ORG-OTO1-TEST
+                admin-c:        TP1-TEST
+                tech-c:         TP1-TEST
+                mnt-by:         RIPE-NCC-END-MNT
+                mnt-by:         LIR-MNT
+                source:         TEST
+                abuse-c:        AH1-TEST
```
Comment (eshryane) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AutNumAuthSpec.groovy]: source should be the last attribute by convention, abuse-c should be moved up

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy [hunk 2]
```diff
@@ -2259,4 +2259,41 @@ class Inet6numSpec extends BaseQueryUpdateSpec {
...
+                descr:        European Regional Registry
+                country:      EU
+                org:          ORG-LIR1-TEST
+                admin-c:      TP1-TEST
+                tech-c:       TP1-TEST
+                mnt-by:       RIPE-NCC-HM-MNT
+                mnt-lower:    RIPE-NCC-HM-MNT
+                status:       ALLOCATED-BY-RIR
+                source:       TEST
+                abuse-c:      AH1-TEST
```
Comment (eshryane) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy]: Move abuse-c up

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AbuseHandlingSpec.groovy [hunk 3]
```diff
@@ -719,18 +719,15 @@ class AbuseHandlingSpec extends BaseQueryUpdateSpec {
...
+        ack.countErrorWarnInfo(0, 0, 1)
         ack.successes.any { it.operation == "Create" && it.key == "[inetnum] 192.168.200.0 - 192.168.200.127" }
         ack.infoSuccessMessagesFor("Create", "[inetnum] 192.168.200.0 - 192.168.200.127") == [
                 "Authorisation override used"]
-        ack.errors.any { it.operation == "Modify" && it.key == "[inetnum] 192.168.200.0 - 192.168.200.127" }
-        ack.errorMessagesFor("Modify", "[inetnum] 192.168.200.0 - 192.168.200.127") ==
-                ["\"abuse-c\" is not valid for this object type"]
 
-        query_object_not_matches("-r -T inetnum 192.168.200.0 - 192.168.200.127", "inetnum", "192.168.200.0 - 192.168.200.127", "abuse-c")
+        query_object_matches("-r -T inetnum 192.168.200.0 - 192.168.200.127", "inetnum", "192.168.200.0 - 192.168.200.127", "abuse-c")
```
Comment (eshryane) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AbuseHandlingSpec.groovy]: Can you create a separate test for the case "not valid for this object type", with a different object type (e.g. route)

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AutNumAuthSpec.groovy (hunk 1 above)
Comment (sanderbuskens) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AutNumAuthSpec.groovy]: Done

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy (hunk 2 above)
Comment (sanderbuskens) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy]: Done

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AbuseHandlingSpec.groovy (hunk 3 above)
Comment (sanderbuskens) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AbuseHandlingSpec.groovy]: Done

**Pull request merged**